src/cairo_runner.rs

```diff
@@ -253,4 +280,29 @@ mod test {
         .unwrap();
         runner.write_output().unwrap();
     }
+
+    #[test]
+    fn add() {
+        let runner = PyCairoRunner::new(
+            "cairo_programs/fibonacci.json".to_string(),
+            "main".to_string(),
+            Some("small".to_string()),
+        )
+        .unwrap();
+        runner.add();
+    }
+
+    #[test]
+    fn get_builtins_initial_stack() {
+        Python::with_gil(|py| {
+            let runner = PyCairoRunner::new(
+                "cairo_programs/fibonacci.json".to_string(),
+                "main".to_string(),
+                Some("small".to_string()),
+            )
+            .unwrap();
+
+            runner.get_builtins_initial_stack(py);
+        });
+    }
```

Comment:
Please add a test that uses a program which uses builtins and check the value of the returned initial stacks